jpa/spi/src/main/java/org/jipijapa/plugin/spi/PersistenceUnitMetadata.java

```diff
@@ -90,4 +90,12 @@ public interface PersistenceUnitMetadata extends PersistenceUnitInfo {
 
     List<ClassTransformer> getTransformers();
 
+    Object getLockObject();
```

Comment:
We shouldn't use this. The `wasDelegatingTransformerAdded` and `setDelegatingTransformerAdded` should handle the locking and not require a consumer to know that.

Reply:
Addressed in change just pushed.